File: HZW.ZHCG2/HZW.ZHCG.WebAPI/app/model/Advert.js
Ext.define('TianZun.model.Advert', {
    extend: 'Ext.data.Model',
    fields: [
        { name: 'ID', type: 'int' },
        {
            name: 'IDType', type: 'string', convert: function (value, record) {
                var idtype = (Array(6).join(0) + record.get('ID')).slice(-6);
                return value + idtype;
            }
        },
        { name: 'TypeID', type: 'int' },
        { name: 'TypeName', type: 'string' },
        { name: 'AdName', type: 'string' },
        { name: 'UnitName', type: 'string' },
        { name: 'UnitPerson', type: 'string' },
        { name: 'UnitPhone', type: 'string' },
        { name: 'Producers', type: 'string' },
        { name: 'Prophone', type: 'string' },
        {
            name: 'State', type: 'string', convert: function (value, record) {
                var state;
                if (new Date(record.get('EndDate')) < new Date()) {
                    state = '<img src="../../Images/已到期.png" />';
                } else if (new Date(record.get('EndDate')) <= new Date((new Date() / 1000 + 86400 * 7) * 1000)) {
                    state = '<img src="../../Images/将到期.png" />';
                } else {
                    state = '<img src="../../Images/未到期.png" />';
                }
                return state;
            }
        },
        { name: 'ExamUnit', type: 'string' },
        {
            name: 'ExamDate', type: 'string', convert: function (value) {
                var examdate = Ext.Date.format(new Date(value), "Y-m-d");
                return examdate;
            }
        },
        {
            name: 'StartDate', type: 'string', convert: function (value) {
                var startdate = Ext.Date.format(new Date(value), "Y-m-d");
                return startdate;
            }
        },
        {
            name: 'EndDate', type: 'string', convert: function (value) {
                var enddate = Ext.Date.format(new Date(value), "Y-m-d");
                return enddate;
            }
        },
        { name: 'Address', type: 'string' },
        { name: 'Photo1', type: 'string' },
        { name: 'Photo2', type: 'string' },
        { name: 'Photo3', type: 'string' },
        { name: 'Photo4', type: 'string' },
        { name: 'FileName1', type: 'string' },
        { name: 'FileName2', type: 'string' },
        { name: 'FileName3', type: 'string' },
        { name: 'FilePath1', type: 'string' },
        { name: 'FilePath2', type: 'string' },
        { name: 'FilePath3', type: 'string' },
        { name: 'Grometry', type: 'string' },
        { name: 'Volume', type: 'number' },
        { name: 'VLong', type: 'number' },
        { name: 'VWide', type: 'number' },
        { name: 'VHigh', type: 'number' },
        { name: 'Materials', type: 'string' },
        { name: 'Curingunit', type: 'string' },
        { name: 'Superviseunit', type: 'string' },
        { name: 'Createtime', type: 'string' },
        { name: 'Createuserid', type: 'int' },
        { name: 'Status', type: 'int' },
        { name: 'Remark', type: 'string' },
    ]
});

File: HZW.ZHCG2/HZW.ZHCG.Model/Advert.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace HZW.ZHCG.Model
{
    public class Advert
    {
        public int ID { get; set; }
        public string IDType { get; set; }
        public int? TypeID { get; set; }
        public string TypeName { get; set; }
        public string AdName { get; set; }
        public string UnitName { get; set; }
        public string UnitPerson { get; set; }
        public string UnitPhone { get; set; }
        public string Producers { get; set; }
        public string Prophone { get; set; }
        public int? State { get; set; }
        public string ExamUnit { get; set; }
        public DateTime? ExamDate { get; set; }
        public DateTime? StartDate { get; set; }
        public DateTime? EndDate { get; set; }
        public string Address { get; set; }
        public string Photo1 { get; set; }
        public string Photo2 { get; set; }
        public string Photo3 { get; set; }
        public string Photo4 { get; set; }
        public string FileName1 { get; set; }
        public string FileName2 { get; set; }
        public string FileName3 { get; set; }
        public string FilePath1 { get; set; }
        public string FilePath2 { get; set; }
        public string FilePath3 { get; set; }
        public string Grometry { get; set; }
        public double? Volume { get; set; }
        public double? VLong { get; set; }
        public double? VWide { get; set; }
        public double? VHigh { get; set; }
        public string Materials { get; set; }
        public string Curingunit { get; set; }
        public string Superviseunit { get; set; }
        public DateTime? Createtime { get; set; }
        public int? Createuserid { get; set; }
        public int? Status { get; set; }
        public string statusName { get; set; }
        public string Remark { get; set; }
    }
}
